### DISCUSSION TITLE
nuxtjs connection

### DISCUSSION THREAD

**Discussion by JackDurden00** [auth0/auth0-vue#471] [Q&A]
### Checklist

- [X] The issue can be reproduced in the [auth0-vue sample app](https://github.com/auth0-samples/auth0-vue-samples/tree/master/01-Login) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/auth0-vue#readme), [Examples](https://github.com/auth0/auth0-vue/blob/main/EXAMPLES.md), and [FAQ](https://github.com/auth0/auth0-vue/blob/main/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/auth0-vue/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/auth0-vue/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

I'm trying to connect with 'auth0-vu'

in a ts file before loading my view.

`import { defineNuxtPlugin } from '@nuxtjs/composition-api' // Utilisation du composition API pour Nuxt 2
import { createAuth0 } from '@auth0/auth0-vue'

export default defineNuxtPlugin((context, inject) => {
  const config = {
    domain: 'dev*********com',
    clientId: 'M3******22H',
    authorizationParams: {
      redirect_uri: `${window.location.origin}/listGroups`,
    },
  }

  const auth0 = createAuth0(config)
  console.log('Auth0 instance created:', auth0)

  //context.app.use(auth0)
  inject('auth0', auth0)
})
`

but auth0 still in loading
and i try to use loginWithRedirect or another function, its still undefined.

i put a loop for wait the loading but it doesn't end

### Reproduction

import { defineNuxtPlugin } from '@nuxtjs/composition-api' // Utilisation du composition API pour Nuxt 2
import { createAuth0 } from '@auth0/auth0-vue'

export default defineNuxtPlugin((context, inject) => {
  const config = {
    domain: 'dev*********com',
    clientId: 'M3******22H',
    authorizationParams: {
      redirect_uri: `${window.location.origin}/listGroups`,
    },
  }

  const auth0 = createAuth0(config)
  console.log('Auth0 instance created:', auth0)

  //context.app.use(auth0)
  inject('auth0', auth0)
})

### auth0-vue version

    "@auth0/auth0-vue": "^2.3.3",     "@nuxtjs/auth": "^4.9.1"

### Vue version

"vue": "^2.7.10",

### Which browsers have you tested in?

Chrome

**Comment by arpit-jn** [comment]
Hi @JackDurden00,

Thanks for your question about using auth0-vue with Nuxt.js and apologies for the delayed response on this.

I'm moving this to our discussions section where it's better suited for community input. The auth0-vue SDK is designed for standard Vue applications, and Nuxt has architectural differences that make integration a topic worth discussing in detail.

In the discussion, it would help if you could share:
- Which Nuxt.js version you're using
- Any specific auth requirements you have
- What you've already tried, if anything

This will help the community provide more targeted guidance on the best approach for your specific use case.
